engine/engine.go

```diff
@@ -181,8 +181,13 @@ func createFiles(paths []*spec.File) error {
 		}
 
 		path := f.Path
+
+		// make the file writable (if it exists)
 		if _, err := os.Stat(path); err == nil {
-			continue
+			if err = os.Chmod(path, 0644); err != nil {
+				return errors.Wrap(err,
```

Comment:
let's just log and continue instead of returning

Reply:
done